go.mod

```
@@ -1,6 +1,6 @@
 module github.com/score-spec/score-compose
 
-go 1.23.5
+go 1.23.6
```

```suggestion
go 1.24
```

Go 1.24 is released (https://go.dev/blog/go1.24). Let's go straight to that? It has a number of major benefits that we can start using soon like the VCS versioning tags when installed/built from tags.

Any reason why we can't go to 1.24?

---

Good point, let's do it.